Week-1/Exercise 1.2/index.html

```diff
@@ -0,0 +1,90 @@
+    <title>Exercise 1.2</title>
+</head>
+<body>
+    <main>
+        <nav class="navbar">
+            <i class="fa-solid fa-bars"></i>
+            <i class="fa-solid fa-house"></i>
+            <i class="fa-solid fa-magnifying-glass"></i>
+            <i class="fa-solid fa-volume-high"></i>
+            <i class="la-volume-up"></i>
+            <i class="fa-solid fa-user"></i>
+            <i class="fa-brands fa-spotify"></i>
+            <i class="fa-solid fa-gear"></i>
+            <i class="fa-brands fa-soundcloud"></i>
+        </nav>
+
+        <section class="primary">
+            <div class="image">
+                <img src="assets/cattyboard.jpg" alt="Album"> 
+            </div>
+            <div class="description">
+                <div class="album">
+                    <p class="title">CattyBoard Top 100 Single Charts(11.06.36) </p>
+                    <p>Unknown Artist</p>
+                    <p>2016.Charts.100 Songs</p>
+                </div>
+              <br/><br/>
+                <div class="features">
+                    <i class="fa-solid fa-play"> Play all</i> 
+                    <i class="fa-solid fa-plus"> Add to</i>
+                    <i class="fa-solid fa-ellipsis"> More</i>
+                </div>
+            </div>
+        </section>
+        <section class="songs">
+            <table>
+              <tr>
+                <td>1.One Dance</td>
+                <td>Crake feat CatKid & Cyla</td>
+                <td>2:54</td>
+                <td class="cloud-sync"><a href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+              <tr>
+                <td>2.Panda</td>
+                <td>Cattee</td>
+                <td>4:06</td>
+                <td class="cloud-sync"><a href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+              <tr>
+                <td>3.Can't Stop the Feeling!</td>
+                <td>Catin Cimberlake</td>
+                <td>3:56</td>
+                <td class="cloud-sync"><a href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+              <tr>
+                <td>4.Working From Home</td>
+                <td>Cat Harmony feat Colla</td>
+                <td>3:34</td>
+                <td class="cloud-sync"><a href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+            </table>
+          </section>
+          <footer class="current-song">
+            <div class="playing-details">
+                <img class="small-img" src="assets/cattyboard.jpg" alt="Album">
+            </div>
+              <div class="info">
+                <div class="album-name">If It Ain't Love</div>
+                <div class="artist-name">Catson Derulo</div>
+              </div>
+              <div class="song-functionality">
```

Comment:
try to create more expressive name. example 
```suggestion
              <div class="song-actions">
```